VariablesHomework/VariablesHomework.cs

```diff
@@ -20,38 +27,48 @@ public void Declaration()
         public void Assignment()
```

Comment:
The Assignment block doesn't pass the tests. At least line 40 fails. (It will only show the first failure in a block).